asv/benchmark.py

```diff
@@ -487,7 +489,7 @@ def _load_vars(self):
         self.number = int(_get_first_attr(self._attr_sources, 'number', 0))
         self.sample_time = _get_first_attr(self._attr_sources, 'sample_time', 0.01)
         self.warmup_time = _get_first_attr(self._attr_sources, 'warmup_time', -1)
-        self.timer = _get_first_attr(self._attr_sources, 'timer', process_time)
+        self.timer = _get_first_attr(self._attr_sources, 'timer', timeit.default_timer)
```

Comment:
is there a reason why you're not already using `wall_timer` here?

Reply:
Nope, will clean up